gradle-plugin/openshift/src/main/java/org/eclipse/jkube/gradle/plugin/OpenShiftExtension.java

```diff
@@ -67,7 +67,7 @@ public ResourceClassifier getResourceClassifier() {
   @Override
   public File getManifest(KitLogger kitLogger, KubernetesClient kubernetesClient, JavaProject javaProject) {
     if (OpenshiftHelper.isOpenShift(kubernetesClient)) {
-      return getOpenShiftManifest().getOrElse(new File(javaProject.getBaseDirectory(), DEFAULT_OPENSHIFT_MANIFEST));
+      return getOpenShiftManifest().getOrElse(new File(javaProject.getOutputDirectory(), DEFAULT_OPENSHIFT_MANIFEST));
```

Comment:
Please, follow a similar approach so that this doesn't fail in Windows:

https://github.com/eclipse/jkube/blob/24da7b57a56d6a954f7dcaba47380b491717a550/gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/KubernetesExtension.java#L418